app/assets/stylesheets/index.scss

```diff
@@ -61,3 +78,30 @@ a {
 .pagination {
   text-align: center;
 }
+
+
+.footer {
+  background-color: #292c2f;
+  text-align: center;
+  font-size: 1.2rem;
+  padding: 2rem; }
+  .footer p.slogan {
+    color: #8d9093;
+    font-size: 1.8rem;
+    margin: 0 0 1rem; }
+  .footer p.copywrite {
+    color: #8f9296;
+    font-size: 0.875rem;
+    margin: 0; }
+  .footer .links a {
+    list-style: none;
+    font-weight: bold;
+    color: #ffffff;
+    padding: 3rem 0 2rem;
+    margin: 0; }
+    .footer .links a::after {
+      content: "•";
+      padding: 0 0.4rem 0 0.8rem; }
+    .footer .links a:last-child::after {
```

Comment:
Line should be indented 0 spaces, but was indented 4 spaces<br>Merge rule `.footer .links a:last-child::after` with rule on line 83<br>Selector should have depth of applicability no greater than 2, but was 3